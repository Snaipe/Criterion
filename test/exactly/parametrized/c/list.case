[act]

% parameterized.c.bin --list

[assert]

`Parameterized tests should be listed as one test`

stdout equals -contents-of expected/list.txt

stderr is-empty
