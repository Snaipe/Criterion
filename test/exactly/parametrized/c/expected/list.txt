params: 4 tests
|-- cleanup
|-- multiple
|-- simple
`-- string
